src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.hpp

```diff
@@ -51,12 +51,23 @@ struct BrgemmKernelConfig : public snippets::KernelExecutorBase::GenericConfig {
 #endif
 
 private:
+    struct StaticParams {
+        StaticParams(const element::Type& in0_dtype, const element::Type& in1_dtype, float beta,
```

Comment:
In one of the previous discussions, you agreed with Alexandra to delete default constructor for safety reasons. Could you please do that?

Reply:
Yes, but we were talking about the `BrgemmKernelConfig` and its default constructor is deleted [here](https://github.com/IvanNovoselov/openvino/blob/eb05ad372bf39b347c1c26c44adf6da469a8a7f6/src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.hpp#L18)
As for `StaticParams`, there is no point is deleting it's default constructor, since the class doesn't hold any pointers, and its perfectly safe to default-initialize it.

Reply:
My bad :) Thanks for the explanation